techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md

```diff
@@ -1,8 +1,8 @@
 # Not Using a Proxy Pass
 
-> ### `Important`
-> This package is set up to use a proxy pass to the backend by default using `/api`.  
-> This means that all traffic to `<frontend url>/api` will be directed to `<backend_url>`.  
+!!! info "Important"
```

Comment:
```suggestion
!!! important
```
Can we try this to see if it works?

Reply:
important preview: 
![information_testpng](https://github.com/bcgov/citz-imb-sso-react/assets/78506153/a9cb4e94-a1ba-4b91-9921-f85c5ffe2085)